data/presets/shop/hairdresser.json

```
@@ -7,6 +7,10 @@
     "terms": [
         "beard"
     ],
+    "fields": [
+        "name",
```

Please also include the regular fields of the `shop` preset:

```suggestion
        "name",
        "operator",
        "address",
        "building_area_yes",
        "opening_hours",
        "payment_multi",
        "phone",
```